### PR TITLE
[full-ci] Show file icon if no thumbnail is present

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7337

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Screenshots (if appropriate):
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10Files4</strong> https://drone.owncloud.com/owncloud/web/27049/20/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIFilesDetails-fileDetails_feature-L42.png
  

### webUIFilesDetails-fileDetails_feature-L57.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by kulmann — APPROVED** [review]
😍

**Comment by SagarGi** [comment]
I will have a look into this. @individual-it

**Comment by JanAckermann** [comment]
@SagarGi 
Please note: In the acceptance tests, we want to check, that the big preview icon is displayed if we are in the details panel,
if we are in a sub panel e.G sharing|actions we want to check, that the small icon is displayed

**Comment by SagarGi** [comment]
> @SagarGi Please note: In the acceptance tests, we want to check, that the big preview icon is displayed if we are in the details panel, if we are in a sub panel e.G sharing|actions we want to check, that the small icon is displayed

@JanAckermann thank you for the information i will be checking them in both (small and big)

**Comment by SagarGi** [comment]
I have refactored some steps and code as per this comment https://github.com/owncloud/web/pull/7344#issuecomment-1199063515 can somebody review the test code @saw-jan @SwikritiT @kiranparajuli589 @grgprarup .

File: tests/acceptance/features/webUIFilesDetails/fileDetails.feature [hunk 1]
```diff
@@ -16,27 +16,32 @@ Feature: User can open the details panel for any file or folder
     And the user has browsed to the personal page
     When the user opens the sidebar for file "lorem.txt" on the webUI
     Then the app-sidebar should be visible
-    And the thumbnail should be visible in the app-sidebar
     And the "details" details panel should be visible
+    And the "big" preview of thumbnail should be visible in the "details" panel
     When the user switches to "actions" panel in details panel using the webUI
     And the "actions" details panel should be visible
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFilesDetails/fileDetails.feature:22]:
```suggestion
    Then the "actions" details panel should be visible
```
I think this should be a then  step

File: tests/acceptance/features/webUIFilesDetails/fileDetails.feature [hunk 2]
```diff
@@ -16,27 +16,32 @@ Feature: User can open the details panel for any file or folder
...
   Scenario: View different areas of the app-sidebar for a folder in files page
     Given user "Alice" has created folder "simple-folder" in the server
     And the user has browsed to the personal page
     When the user opens the sidebar for folder "simple-folder" on the webUI
     Then the app-sidebar should be visible
-    And the thumbnail should be visible in the app-sidebar
     And the "details" details panel should be visible
+    And the "big" preview of thumbnail should be visible in the "details" panel
     When the user switches to "actions" panel in details panel using the webUI
     And the "actions" details panel should be visible
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFilesDetails/fileDetails.feature:40]:
```suggestion
    Then the "actions" details panel should be visible
```

File: tests/acceptance/features/webUIFilesDetails/fileDetails.feature [hunk 3]
```diff
@@ -46,12 +51,17 @@ Feature: User can open the details panel for any file or folder
     And the user has browsed to the favorites page
     When the user opens the sidebar for file "lorem.txt" on the webUI
     Then the app-sidebar should be visible
-    And the thumbnail should be visible in the app-sidebar
     And the "details" details panel should be visible
+    And the "big" preview of thumbnail should be visible in the "details" panel
     When the user switches to "actions" panel in details panel using the webUI
     And the "actions" details panel should be visible
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFilesDetails/fileDetails.feature:57]:
```suggestion
    Then the "actions" details panel should be visible
```

File: tests/acceptance/features/webUIFilesDetails/fileDetails.feature [hunk 4]
```diff
@@ -61,14 +71,14 @@ Feature: User can open the details panel for any file or folder
     And the user has browsed to the favorites page
     When the user opens the sidebar for folder "simple-folder" on the webUI
     Then the app-sidebar should be visible
-    And the thumbnail should be visible in the app-sidebar
     And the "details" details panel should be visible
+    And the "big" preview of thumbnail should be visible in the "details" panel
     When the user switches to "actions" panel in details panel using the webUI
     And the "actions" details panel should be visible
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFilesDetails/fileDetails.feature:77]:
```suggestion
   Then the "actions" details panel should be visible
```

File: tests/acceptance/features/webUIFilesDetails/fileDetails.feature [hunk 5]
```diff
@@ -78,7 +88,7 @@ Feature: User can open the details panel for any file or folder
       | path | simple-folder |
     And the user has browsed to the shared-via-link page
     When the user opens the actions sidebar panel of folder "simple-folder" on the webUI
-    Then the thumbnail should be visible in the app-sidebar
+    And the "small" preview of thumbnail should be visible in the "actions" panel
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFilesDetails/fileDetails.feature:91]:
```suggestion
    Then the "small" preview of thumbnail should be visible in the "actions" panel
```

File: tests/acceptance/features/webUIFilesDetails/fileDetails.feature [hunk 6]
```diff
@@ -61,14 +71,14 @@ Feature: User can open the details panel for any file or folder
...
     Then the app-sidebar should be visible
-    And the thumbnail should be visible in the app-sidebar
     And the "details" details panel should be visible
+    And the "big" preview of thumbnail should be visible in the "details" panel
     When the user switches to "actions" panel in details panel using the webUI
     And the "actions" details panel should be visible
+    And the "small" preview of thumbnail should be visible in the "actions" panel
     When the user switches to "people" panel in details panel using the webUI
     Then the "people" details panel should be visible
-    When the user switches to "links" panel in details panel using the webUI
```
Comment (SwikritiT) [inline tests/acceptance/features/webUIFilesDetails/fileDetails.feature:70]: Why remove this step?

Comment (SagarGi) [reply tests/acceptance/features/webUIFilesDetails/fileDetails.feature:70]: @SwikritiT this steps works same as `When the user switches to "people" panel in details panel using the webUI` when observing. so i removed it and instead checked the version panel which was missing.

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md [hunk 7]
```diff
@@ -299,8 +299,8 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIUpload/upload.feature:42](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/upload.feature#L42)
 
 ### [Favorites deactivated in ocis temporarily](https://github.com/owncloud/ocis/issues/1228)
--   [webUIFilesDetails/fileDetails.feature:42](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesDetails/fileDetails.feature#L42)
--   [webUIFilesDetails/fileDetails.feature:57](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesDetails/fileDetails.feature#L57)
+-   [webUIFilesDetails/fileDetails.feature:47](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesDetails/fileDetails.feature#L42)
+-   [webUIFilesDetails/fileDetails.feature:67](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesDetails/fileDetails.feature#L57)
```
Comment (kiranparajuli589) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:303]: 47, 42
67, 57
path and link has different line nummbers

**Comment by SagarGi** [comment]
@JanAckermann i might have messed up while rebasing and resolving. can you have a look into it?

File: tests/acceptance/features/webUIFilesDetails/fileDetails.feature (hunk 6 above)
Comment (SwikritiT) [reply tests/acceptance/features/webUIFilesDetails/fileDetails.feature:70]: @SagarGi Okay. I think there's still that step in other scenarios too. Can you check if they can be remove them as well?

Comment (SagarGi) [reply tests/acceptance/features/webUIFilesDetails/fileDetails.feature:70]: A separate issue can be created for it. i will make a separate issue to check and remove it from scenarios.

**Review by SwikritiT — COMMENTED** [review]
tests LGTM

**Comment by SagarGi** [comment]
@JanAckermann Please have a look into it.

**Comment by JanAckermann** [comment]
@SagarGi LGTM, THX!

**Pull request merged**